### PR TITLE
Add prometheus metrics 

### LINKED ISSUE #142: Include prometheus metrics of false positives
**Is your feature request related to a problem? Please describe.**
Provide metrics to collect the following values:

- num of anomalies found  = anomaly_count
- num of log lines processed = log_lines_processed
- num of false positives reported = num_false_positives
- Anomaly score for false anomaly (id/score) = false_positive_id_score
- Training time = training_time

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by zmhassan**
## Description

Metrics to collect:
- num of anomalies found  = anomaly_count
- num of log lines processed = log_lines_processed
- num of false positives reported = num_false_positives
- Anomaly score for false anomaly (id/score) = false_positive_id_score
- Training time = training_time

Fixes
#142

### CODE REVIEW

**Comment by zmhassan** [comment]
These metrics already exist as was added by Vaclav prior:
- num of anomalies found = anomaly_count
- num of log lines processed = log_lines_processed
- num of false positives reported = num_false_positives
- Training time = training_time

I only need to add this one:
 - Anomaly score for false anomaly (id/score) = false_positive_id_score

**Comment by durandom** [comment]
@zmhassan please fix the conflicting files

**Comment by zmhassan** [comment]
Fixed merge conflict

**Comment by zmhassan** [comment]
> @zmhassan please fix the conflicting files

It's fixed now.

File: anomaly_detector/anomaly_detector.py [hunk 1]
```diff
@@ -187,8 +190,13 @@ def infer(self, false_positives=None):
                 s["predict_id"] = str(uuid.uuid4())
                 s["anomaly_score"] = dist[i]
                 # Record anomaly event in fact_store and
+                if false_positives is not None:
+                    if {"message": s["message"]} in false_positives:
+                        _LOGGER.info("False positive was found (score: %f): %s" % (dist[i], s["message"]))
+                        FALSE_POSITIVE_METRIC.labels(id=s["predict_id"], message=s["message"], score=dist[i]).inc()
```
Comment (durandom) [inline anomaly_detector/anomaly_detector.py]: @zmhassan just remove the message and score label from the metric. Then it's just a counter of reported `FALSE_POSITIVES`

**Comment by durandom** [comment]
also rebase on top of master to get rid of the merged commits

File: anomaly_detector/anomaly_detector.py (hunk 1 above)
Comment (zmhassan) [reply anomaly_detector/anomaly_detector.py]: Good point.

Comment (zmhassan) [reply anomaly_detector/anomaly_detector.py]: Fixed it. Done !

**Comment by zmhassan** [comment]
@durandom its rebased

**Comment by zmhassan** [comment]
@durandom its squashed and rebased.

**Pull request merged**